tests/org.eclipse.fordiac.ide.test.library/src/org/eclipse/fordiac/ide/test/library/LibraryAssert.java

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
+ * Copyright (c) 2024 Primetals Technologies Austria GmbH
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Patrick Aigner - initial API and implementation and/or initial documentation
+ *******************************************************************************/
+package org.eclipse.fordiac.ide.test.library;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+import org.eclipse.core.resources.IProject;
+import org.eclipse.fordiac.ide.library.LibraryManager;
+import org.eclipse.fordiac.ide.library.model.library.Manifest;
+import org.eclipse.fordiac.ide.library.model.util.ManifestHelper;
+import org.eclipse.fordiac.ide.library.model.util.VersionComparator;
+import org.eclipse.fordiac.ide.model.typelibrary.TypeLibraryManager;
+
+public class LibraryAssert {
+	private LibraryAssert() {
+	}
+
+	public static void assertDependencyLinked(final IProject project, final Manifest manifest,
+			final String symbolicName, final String version, final int index) {
+		assertLibraryLinked(project, symbolicName, version);
+		assertNotNull(manifest.getDependencies());
+		assertNotNull(manifest.getDependencies().getRequired().get(index));
+		assertNotNull(symbolicName, manifest.getDependencies().getRequired().get(index).getSymbolicName());
```

Comment:
This feels like it should be a assertEquals instead of assertNotNull?

Reply:
thanks for catching that